docker-compose-flask-microservices.yml

```diff
@@ -3,25 +3,29 @@ version: '3.8'
 services:
   place-service:
     build: Flask/microservices/place-service/
-    container_name: place-service
+    image: flask-place-service-image
+    container_name: flask-place-service-container

```

Comment:
Probably adding `container_name` has an impact on failing tests.
Services have hostnames mapped to container names.
So if you don't need this parameter for now, I think you should remove it from docker-compose files

Reply:
Ok, I'll revert to the original nomenclature in that case.